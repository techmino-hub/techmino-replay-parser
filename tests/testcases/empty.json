{
    "replay": "eJydUt9LwzAQfvevkDwfW9J2Wds3FXwSJ3OKMGRkbdaGtc1IsqmM/e9euqa+S6D57vvuLvejZ+KEfbOyJPlONFYCOUljle5ITt7phM0nLCVAWo0O688eSJTswajObRhtUCyF81xEo2RK0ymLb1maU55HGYqHRvxIg/LzYrW52ywX94sV0lb6J2POE8Z5mnnCOdVVJD+TTn67F21J7swRC7KFNjIYO1EgXlP458EW7F5hd2sGc2AMYmAJJJCBtyPgwCheLIaZ52YoXZ16lkPqCer1CFJMVsjO+fYYkG2ji32ocyuqJ9WNZXvp8YPk1E8LO4uBVLW2rg+0Zc9Rj4QxV9Rq7eoQLlyIrlVVP4g2CF/CdAFjkuLoeq/lKw58hV98xigcNJ0kmLMWeznksbgXWw9G0UhhBtzqU/ApjT6EjKoe96HMH2zt+N947yHQ4QLH0vuGLpebX+txrWE=",
    "expected": {"metadata":{"tasUsed":false,"version":"V0.17.18","mod":[],"mode":"sprint_10l","date":"2024/08/13 18:06:29","player":"NOT_A_ROBOT","seed":366416689,"setting":{"nextPos":true,"score":true,"face":[0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0],"skin":[1,7,11,3,14,4,9,1,7,2,6,10,2,13,5,9,15,4,11,3,10,2,16,8,4,10,13,2,8],"center":1,"block":true,"bagLine":true,"lockFX":0,"das":3,"ghost":1,"sddas":0,"sdarr":0,"smooth":true,"atkFX":0,"highCam":true,"warn":true,"dascut":0,"RS":"TRS","grid":0.4,"shakeFX":0,"splashFX":0,"clearFX":0,"moveFX":0,"dropcut":0,"ihs":true,"irs":true,"ims":false,"dropFX":0,"text":true,"arr":0}},"inputs":[]}
}