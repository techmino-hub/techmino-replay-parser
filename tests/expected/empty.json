{"tasUsed":false,"version":"V0.17.18","mod":[],"mode":"sprint_10l","date":"2024/08/13 18:06:29","player":"NOT_A_ROBOT","seed":366416689,"setting":{"nextPos":true,"score":true,"face":[0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0],"skin":[1,7,11,3,14,4,9,1,7,2,6,10,2,13,5,9,15,4,11,3,10,2,16,8,4,10,13,2,8],"center":1,"block":true,"bagLine":true,"lockFX":0,"das":3,"ghost":1,"sddas":0,"sdarr":0,"smooth":true,"atkFX":0,"highCam":true,"warn":true,"dascut":0,"RS":"TRS","grid":0.4,"shakeFX":0,"splashFX":0,"clearFX":0,"moveFX":0,"dropcut":0,"ihs":true,"irs":true,"ims":false,"dropFX":0,"text":true,"arr":0},"inputs":[]}